### PR TITLE
Allow Invert RC to draw booster

### PR DESCRIPTION
**Pull request by karst**

### CODE REVIEW

**Comment by Gymnasiast** [comment]
How much did you need to change compared to the equivalent Flying RC drawing function? This might be a good opportunity to try creating unified track drawing code (don’t do anything yet, I’m still investigating).

**Comment by karst** [comment]
For drawing barely anything. Both sprite drawing and the bound box offset was upped by 5px. The invert is default 5 or 6px above the flying rc. If I am gonna be completely honest I think it'd be dope to have all BM drawing code be from the same track type file.

I did need to change a parameter in the meta to actually get the boosters to boost. It was set to 0 acceleration. I tried 68 (equal to the twister) but that's waay to fast. The flying rc was set to 25 which was too slow which i was why i chose somewhere in between (38). It's really funny to change that to a random bigger nummer btw, but i digress.

**Comment by karst** [comment]
What snakes have been removed in the past few days?

**Comment by rik-smeets** [comment]
> What snakes have been removed in the past few days?

The past couple of days, I don't think anything? In this PR some of the removed snakes in the Paint code haven't been processed yet. For example, `paint_session` is now `PaintSession`.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - Improved: [#18192] Tycoon Park has been added Extras tab.
 - Improved: [#18214] Competition scenarios have received their own section.
 - Improved: [#18250] Added modern style file and folder pickers on Windows.
+- Improved: [#18332] Allow Invert RC to draw booster.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:7]:
```suggestion
- Improved: [#18332] Allow Inverted Roller Coaster to draw boosters.
```

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**